modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="alibaba-ai-converting-image-to-qcow2_{context}"]
+= Converting the Discovery image to QCOW2 format
+
+Convert the generated ISO to `QCOW2` format before importing it into {alibaba}.
+
+.Prerequisites
+
+* You have created a cluster and downloaded the Discovery image in {ai-full}.
+
+.Procedure
+
+. Open the command line interface. 
+
+. Verify that `qemu-img` is installed:
+
+.. On {op-system-base-full} or {op-system-base}, run this command:
++
+[source,terminal]
+----
+$ sudo dnf install -y qemu-img
+----
+
+.. On Ubuntu, run this command:
```

Comment:
Hi @faermanj Question from John Wilkins - are we still supporting Ubuntu or should we take that out?

Reply:
Just a gentle reminder that you will need SME/QE review as well as peer review. They can be requested simultaneously.